examples/access-control-migration/src/migrate.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
```

Comment:
why do we need a migrate.ts file for a sample?

Reply:
copied from todo example :p, removed.